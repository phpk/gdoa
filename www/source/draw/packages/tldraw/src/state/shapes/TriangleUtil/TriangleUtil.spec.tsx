import { Triangle } from '..'

describe('Triangle shape', () => {
  it('Creates a shape', () => {
    expect(Triangle.create({ id: 'triangle' })).toMatchSnapshot('triangle')
  })
})
