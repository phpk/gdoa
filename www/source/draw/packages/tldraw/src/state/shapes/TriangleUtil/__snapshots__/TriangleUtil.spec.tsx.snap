// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Triangle shape Creates a shape: triangle 1`] = `
Object {
  "childIndex": 1,
  "id": "triangle",
  "name": "Triangle",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "size": Array [
    1,
    1,
  ],
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": false,
    "scale": 1,
    "size": "small",
  },
  "type": "triangle",
}
`;
